client/src/components/Header.tsx

```diff
@@ -7,7 +6,7 @@ export default function Header( {text, backPath}: Prop ){
     return(
         <>
             <div className="flex flex-row align-middle my-6 mx-3 font-mono">
-                <Link href={backPath}><Image src={back_arrow} alt="arrow" /></Link>
+                <Link href={backPath}><Image src={'/images/back_arrow.png'} width={32} height={32} alt="arrow" /></Link>
```

Comment:
src does not need to be wrapped inside {}. This is just a string, no need to include typescript execution in it. Same for width and height.

Write it as you would write a simple HTML file.